README.md

```diff
@@ -3,14 +3,31 @@
 <a href="https://doi.org/10.5281/zenodo.4296287"><img src="https://zenodo.org/badge/DOI/10.5281/zenodo.4296287.svg" alt="DOI"></a>
 <a href="https://github.com/bayesiains/nflows/actions/workflows/build_lint_test.yml"><img src="https://github.com/bayesiains/nflows/actions/workflows/build_lint_test.yml/badge.svg" alt="Build status"></a>
 <a href="https://opensource.org/licenses/MIT"><img src="https://img.shields.io/badge/License-MIT-green.svg"></a>
+<a href="https://github.com/conda-forge/nflows-feedstock"><img src="https://img.shields.io/conda/vn/conda-forge/nflows.svg"></a>
 
 `nflows` is a comprehensive collection of [normalizing flows](https://arxiv.org/abs/1912.02762) using [PyTorch](https://pytorch.org).
 
 ## Installation
 
+### From PyPI
+
 To install from PyPI:
 ```
-pip install nflows
+python -m pip install nflows
```

Comment:
Why is this preferable to the more common ‘pip install’?

Reply:
This pattern is the only way to make sure that the pip you want is the pip you get, as even in a virtual environment it is possible to mess up paths and get a pip elsewhere which is not the pip tied to the python that created the environment. It is also the recommended pattern by the pip and Python docs. Example: https://docs.python.org/3/installing/index.html